### DISCUSSION TITLE
Micro Machines 2 Turbo Tournament Mega Drive/Genesis Rom won't play after dump

### DISCUSSION THREAD

**Discussion by benmerrick3d** [sanni/cartreader#504] [Need help]
This is the last of my cartridges that I can't get a working rom out of. I have managed to dump the rest of my Mega Drive carts fine, but this special J Cart won't work:

https://www.giantbomb.com/j-cart/3015-1234/

It seems to dump okay, and I have successfully got the save off the cartridge (works when tested after), but the rom itself doesn't work when tested.

Has anybody managed to get this rom, or another j cart, successfully dumped? Any tips on what I could try?
I have the V3 cart reader from Save The Hero Builders without the clock gen. I'd love to get to the bottom of it, thanks

**Comment by PsyK0p4T** [comment]
Can you post a picture of your cart ? just to confirm the region.
What is the CRC32 of the rom you obtained ?

**Reply by benmerrick3d** [reply]
![20220901_154900](https://user-images.githubusercontent.com/30409503/187944335-62bd1466-ef58-40a8-819e-c66e449fc984.jpg)
![20220901_154912](https://user-images.githubusercontent.com/30409503/187944343-3e602008-b088-4cbf-adfb-b96d0f5892e8.jpg)

Bought and used in UK, so I assume UK and/or Europe. 
Crc32 D7DE473E (In the text output file it says 'Not found' after the number)

**Reply by PsyK0p4T** [reply]
Yup, European version here. So, the correct CRC for this J-cart should be "42bfb7eb".
If you get a different one, it could me that the cart is not read perfectly, or that the dumper can handle the cart correctly (Codemasters ones are a bit different than SEGA ones).

Have you tried to hard clean the connectors with q-tips + alcohol ?
Do you always obtain the same CRC, or a different one each time you insert/redump the cart ?

**Reply by benmerrick3d** [reply]
Thanks for the tips. My carts are very clean, but I cleaned it again, nothing came off.

Ran it twice with 5V, one with eep one without (as it has eep storage) and both times it gave: Crc32... D7DE473E - - > Not found

**Reply by benmerrick3d** [reply]
Is it definitely the cart giving the wrong Crc32 or could the software library on sd card be wrong? Is it worth trying an older version of the software?

**Reply by benmerrick3d** [reply]
I've also tried lifting the cart slightly, moving it slightly to one side or the other. It always reads, and always gives the same crc32

**Reply by PsyK0p4T** [reply]
yes, keep the 5v switch, and all others stay in left position.
crc "D7DE473E" doen't exist in DoM database, and I doubt you have a new revision in hands so... high chance this is a bad dump, and not working in emulators due to this.

Having a defective cart is still possible, but really rare.
Most of time it is due to connectors (cartridge, or cart reader ones) which are not perfectly clean, but you would have a different crc everytime.
Maybe the current code can't handle that title correctly. Which is very possible, as Codemasters cartridges are particular.
Could be cool if someone else having the game could test, and share the results here.

**Reply by benmerrick3d** [reply]
Thanks for your thoughts. I really hope someone else replies who has the game to test

**Comment by benmerrick3d** [comment]
Does anybody else have this game and has tried to dump it? I'd love to work out if it's possible.

Thank you
